buildpacks/nodejs-engine/tests/integration_test.rs

```diff
@@ -0,0 +1,59 @@
+#[cfg(test)]
```

Comment:
```suggestion
```

As far as I'm aware, tests under `tests/` (integration tests) don't need to be guarded under a `cfg` attribute, since they won't be compiled unless using `--all-targets` anyway?

Reply:
Oh also, you'll want a `#![warn(clippy::pedantic)]` here:

```suggestion
#![warn(clippy::pedantic)]
```

Reply:
I think I was using `#[cfg(test)]` to attempt to get rid of `unused_crate_dependencies` errors during tests. It didn't work =P.